index.html

```diff
@@ -6,9 +6,224 @@
+          architecto beatae vitae dicta sunt explicabo.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="design-principles color-block">
+    <h1>Design principles</h1>
+
+    <div class="row">
+      <div class="col-xs-12 col-sm-6">
+        <h2>Technological Neutrality</h2>
+        <span>Supporting content from popular social media platforms along with media originating from pur- pose-built systems.</span>
+      </div>
+
+      <div class="col-xs-12 col-sm-6">
+        <h2>Computer-enabled Expert Analysis</h2>
+        <span>Automated computer analysis augmenting and enhancing expert human real-time reasoning and decision making.</span>
+      </div>
+
+      <div class="col-xs-12 col-sm-6">
+        <h2>Real-time Response</h2>
+        <span>Moving from online report aggregation to analysis, escalation and response within one hour.</span>
+      </div>
+
+      <div class="col-xs-12 col-sm-6">
+        <h2>Big Data</h2>
+        <span>Supporting up to 1,000 incoming reports per second.</span>
+      </div>
+
+      <div class="col-xs-12 col-sm-6">
+        <h2>Open Source</h2>
+        <span>Aggie is fully open source and welcomes contri- butions.</span>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <h1>Featured Deployment</h1>
+
+    <div class="media row">
+      <div class="col-xs-offset-2 col-sm-offset-0 col-xs-8 col-sm-3">
+        <a href="#">
+          <img class="media-object" src="img/deployments/hate-speech.png"
+            alt="Screenshot of table with short description of harassment incidents and locations">
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-9">
+        <h2 class="media-heading">Hate Speech</h4>
+        <p>
+          Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+          laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+          architecto beatae vitae dicta sunt explicabo.
+        </p>
+      </div>
+    </div>
+
+    <div class="media row">
+      <div class="col-xs-offset-2 col-sm-offset-0 col-xs-8 col-sm-3">
+        <a href="#">
+          <img class="media-object" src="img/deployments/argentina-2015.png"
+            alt="Group of people around a table looking at different computer monitors">
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-9">
+        <h2 class="media-heading">Argentina 2015</h4>
+        <p>
+          Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+          laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+          architecto beatae vitae dicta sunt explicabo.
+        </p>
+      </div>
+    </div>
+
+    <div class="media row">
+      <div class="col-xs-offset-2 col-sm-offset-0 col-xs-8 col-sm-3">
+        <a href="img/deployments/ghana-2012.png" data-toggle="lightbox">
```

Comment:
Clicking on the image opens up the image in a lightbox-like container. This functionality can be removed in the Deployments section, if desired.